### PR TITLE
Suppress edition manager

### PR DESCRIPTION
**Pull request by nicolasThal**

### CODE REVIEW

**Comment by Thiblef** [comment]
Many ternary operators spotted but it stays understandable 
:+1:

File: BackofficeBundle/Security/Authorization/Voter/NodeGroupRoleVoter.php [hunk 1]
```diff
@@ -2,127 +2,13 @@
...
-use OpenOrchestra\ModelInterface\Model\ReadSiteInterface;
-use OpenOrchestra\ModelInterface\Repository\NodeRepositoryInterface;
-use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
-use Symfony\Component\Security\Core\Authorization\Voter\VoterInterface;
+use OpenOrchestra\Backoffice\Security\Authorization\Voter\NodeGroupRoleVoter as BaseNodeGroupRoleVoter;
 
 /**
  * Class NodeGroupRoleVoter
+ *
+ * @deprecated use NodeGroupRoleVoter from Backoffice folder, will be removed in 1.1.2
```
Comment (itkg-alavieille) [inline BackofficeBundle/Security/Authorization/Voter/NodeGroupRoleVoter.php]: in 1.2.0 ?

**Comment by itkg-alavieille** [comment]
else php doc :+1:

File: ApiBundle/Transformer/NodeTransformer.php [hunk 2]
```diff
@@ -46,7 +44,6 @@ public function __construct(
         SiteRepositoryInterface $siteRepository,
         StatusRepositoryInterface $statusRepository,
         EventDispatcherInterface $eventDispatcher,
-        AuthorizeEditionManager $authorizeEdition,
         AuthorizationCheckerInterface $authorizationChecker
     )
     {
```
Comment (itkg-ngilain) [inline ApiBundle/Transformer/NodeTransformer.php]: ``)`` and ``{`` should be on the same line to be PSR2 compliant

**Comment by itkg-ngilain** [comment]
:+1:

**Pull request merged**